sdk/keyvault/azkeys/client.go

```diff
@@ -335,6 +335,67 @@ func (c *Client) CreateRSAKey(ctx context.Context, name string, options *CreateR
 	return createRSAKeyResponseFromGenerated(resp), nil
 }
 
+type CreateOKPKeyOptions struct {
+	// Hardware Protected OCT Key
+	HardwareProtected bool
+
+	// The key size in bits. For example: 2048, 3072, or 4096 for RSA.
+	KeySize *int32 `json:"key_size,omitempty"`
+
+	// The public exponent for a RSA key.
+	PublicExponent *int32 `json:"public_exponent,omitempty"`
+
+	// Application specific metadata in the form of key-value pairs.
+	Tags map[string]*string `json:"tags,omitempty"`
```

Comment:
Why a reference for the map value? Just so values can be `nil`? I don't think this is actually possible service-side.

Reply:
Had a discussion with @jhendrixMSFT about this, we're changing all `Tags map[string]*string` to `Tags map[string]string`, the nil value is not valid.

Reply:
This was taken care of in #16858